backend/src/Mutation.js

```diff
@@ -0,0 +1,10 @@
+const Mutations = {
+  async createUser(args, context, info) {
```

Comment:
You should define function params in this order: `async createUser(info, args, context)` so the request can be handled properly without internal server errors